fondat/aws/bedrock/resources/sessions.py

```diff
@@ -0,0 +1,289 @@
+        invocationStepTime: datetime,
+        *,
+        invocationStepId: str | None = None,
+    ) -> Mapping[str, Any]:
+        """Add invocation step.
+
+        Args:
+            sessionIdentifier: Session identifier
+            invocationIdentifier: Invocation identifier
+            payload: Step payload
+            invocationStepTime: Step timestamp
+            invocationStepId: Step identifier
+
+        Returns:
+            Step details
+        """
+        params = {
+            "sessionIdentifier": sessionIdentifier,
+            "invocationIdentifier": invocationIdentifier,
+            "payload": payload,
+            "invocationStepTime": invocationStepTime,
+        }
+        if invocationStepId is not None:
+            params["invocationStepId"] = invocationStepId
+        async with runtime_client(self.config_runtime) as client:
+            return await client.put_invocation_step(**params)
+
+    @operation(method="get", policies=lambda self: self.policies)
+    async def get_step(
+        self,
+        sessionIdentifier: str,
+        invocationIdentifier: str,
+        invocationStepId: str,
+    ) -> Mapping[str, Any]:
+        """Get invocation step details.
+
+        Args:
+            sessionIdentifier: Session identifier
+            invocationIdentifier: Invocation identifier
+            invocationStepId: Step identifier
+
+        Returns:
+            Step details
+        """
+        async with runtime_client(self.config_runtime) as client:
+            return await client.get_invocation_step(
+                sessionIdentifier=sessionIdentifier,
+                invocationIdentifier=invocationIdentifier,
+                invocationStepId=invocationStepId,
+            )
+
+    @operation(method="get", policies=lambda self: self.policies)
+    async def list_steps(
+        self,
+        sessionIdentifier: str,
+        invocationIdentifier: str | None = None,
+        *,
+        max_results: int | None = None,
+        cursor: str | None = None,
+    ) -> Any:
+        """List invocation steps.
+
+        Args:
+            sessionIdentifier: Session identifier
+            invocationIdentifier: Invocation identifier
+            max_results: Maximum number of results to return
+            cursor: Pagination cursor
+
+        Returns:
+            Page of step summaries
+        """
+        params = {"sessionIdentifier": sessionIdentifier}
+        if invocationIdentifier is not None:
+            params["invocationIdentifier"] = invocationIdentifier
+        if max_results is not None:
+            params["maxResults"] = max_results
+        if cursor is not None:
+            params["nextToken"] = cursor
+        async with runtime_client(self.config_runtime) as client:
+            resp = await client.list_invocation_steps(**params)
+        return paginate(resp, items_key="invocationStepSummaries")
+
+    @operation(method="get", policies=lambda self: self.policies)
+    async def list(
```

Comment:
Shouldn't we be consistent here and use `get` and not have to specify the operation's name. I mean it is really a get on a collection of items, isn't it ?